src/main/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/FileNameHelper.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.slc.services.steps.getpdf;
+
+import com.microsoft.applicationinsights.core.dependencies.apachecommons.io.FilenameUtils;
+import uk.gov.hmcts.reform.slc.model.Letter;
+
+public final class FileNameHelper {
+
+    private static final String separator = "_";
```

Comment:
static final args should be all capitals unless it's logger

Reply:
checkstyle and/or PMD disagree :)